packages/protocol/src/Chief.sol

```diff
@@ -22,12 +25,60 @@ import {CoreRoles} from "./access/CoreRoles.sol";
 contract Chief is CoreRoles, AccessControl, IChief {
   using Address for address;
 
-  event OpenVaultFactory(bool state);
+  /**
+   * @dev Emitted when the deployements of new vaults is alllowed/disallowed.
+   *
+   * @param allowed `true` to allow, `false` to disallow
+   */
+  event AllowPermissionlessDeployments(bool allowed);
+
+  /**
+   * @dev Emitted when a new vault gets deployed.
+   *
+   * @param factory address of the factory through which to deploy
+   * @param vault address of the newly deployed vault
+   * @param deployData encoded args used to deploy the vault
+   */
   event DeployVault(address indexed factory, address indexed vault, bytes deployData);
+
+  /**
+   * @dev Emitted when `_vaults` are set through `setVaults()`.
```

Comment:
should we change to {IChief-setVaults} ?

Reply:
I thought this notation was to be used only when referring to another contract. Here `setVaults()` is a method from the same one. But I might be wrong, @DaigaroCota can you confirm plz?

Reply:
`setVaults()` is a method in the same Chief contract. Therefore there shouldn't be the need to self refer to it as {Chief-setVaults}.